bundle/Resources/config/services.yml

```diff
@@ -80,3 +80,17 @@ services:
             - "@?router.request_context"
         tags:
             - { name: router, priority: 200 }
+
+    netgen.ezplatform_site.param_converter.location:
+        class: Netgen\Bundle\EzPlatformSiteApiBundle\Converter\ContentParamConverter
+        arguments:
+            - '@netgen.ezplatform_site.site'
```

Comment:
@MarioBlazek This should've been the load service instead of site.

I'm getting: `Type error: Argument 1 passed to Netgen\Bundle\EzPlatformSiteApiBundle\Converter\SiteParamConverter::__construct() must implement interface Netgen\EzPlatformSiteApi\API\LoadService, instance of Site_221043a given`